support-frontend/app/controllers/PaperSubscriptionController.scala

```diff
@@ -38,7 +38,7 @@ class PaperSubscription(
   def paper(): Action[AnyContent] = CachedAction() { implicit request =>
     implicit val settings: AllSettings = settingsProvider.getAllSettings()
     val canonicalLink = Some(buildCanonicalPaperSubscriptionLink())
-    val queryPromos = request.queryString.get("promoCode").map(_.toList).getOrElse(Nil)
+    val queryPromos = DefaultPromotions.Paper.june21Promotion :: request.queryString.get("promoCode").map(_.toList).getOrElse(Nil)
```

Comment:
Add the default promotion to the subs page

Reply:
need to rename `queryPromos` as it's not just query string ones any more